src/crypto/CHIPCryptoPAL.h

```diff
@@ -471,7 +471,7 @@ class P256Keypair : public P256KeypairBase
     /** Release resources associated with this key pair */
     void Clear();
 
-private:
+protected:
```

Comment:
Note: not for this PR. 

It feels, the PSA implementation would benefit from not having all these fields duplicated. Of course that would mean that `Pubkey()` either accepts an output parameter or return an object.

Also, does it makes sense to implement the `Serialize` and `Deserialize` methods for PSA. It would be better if the private key does not leaving the secure side.

Reply:
Yes, probably `P256Keypair` interface could be optimized as not all members seem to be used for all OperationalKeystore implementations. When it comes to `Serialize` and `Deserialize`, I don't want to remove it now to still allow for using `PersistentStorageOperationalKeystore` - I would like to allow for switching to PSA even in existing products that have already written NOCs into non-secure storage. But perhaps, we can make it configurable as indeed these methods shouldn't be needed for new products.